grazer/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -10,105 +10,120 @@
+	            				style: {
+	            					'background-color': '#7bafa6',
+	            					'width': nodeSize,
+	            					'height': nodeSize,
+	            					'label': 'data(label)'
+	            				}
+						},
+						{
+							selector: 'edge',
+							style: {
+								'width': 2,
+								'line-color': '#b0c7c3',
+								'target-arrow-color': '#b0c7c3',
+								'target-arrow-shape': 'triangle'
+							}
+						}
+					],
 
-	            		  layout: {
-	            		    name: 'cose-bilkent',
-	            		    nodeDimensionsIncludeLabels: true,
-	            		  }
-	            	});
+					layout: {
+						name: 'cose-bilkent',
+						nodeDimensionsIncludeLabels: true,
+					}
+				});
 	            
-	            cy.on('tap', 'node', function(){
-	            	  window.location.href = "concept/" + this.data('id');
-	            	})
-	            
-	            	cy.ready(function() {
-                    $(".person-entry").hover(highligthPersonInGraph, removeHighlight);
-                });
+				cy.on('tap', 'node', function(){
+					window.location.href = "concept/" + this.data('id');
+				})
+	            cy.ready(function() {
+					$(".person-entry").hover(highligthPersonInGraph, removeHighlight);
+				});  
             	}
-        },
-        error: function() {
-        	   $("#spinner").hide();
-            $("#network").append("Sorry, could not load network.")
-        }
-    });
+		},
+		error: function() {
+			$("#spinner").hide();
+			$("#network").append("Sorry, could not load network.")
+		}
+	});
 	
 	function highligthPersonInGraph() {
 		var id = $(this).data("concept-id");
 		var node = cy.getElementById(id);
-		cy.animate({
-            fit: {
-                eles: node,
-                padding: 230,
-            }
-        });
 		node.animate({
 			css: { 'width': highlightSize, 'height' : highlightSize},
-		});
-		
+		});	
 	}
+	
 	function removeHighlight() {
 		var id = $(this).data("concept-id");
-        var node = cy.getElementById(id);
-        node.animate({
-            css: { 'width': nodeSize, 'height' : nodeSize}
-        });
+		var node = cy.getElementById(id);
+		cy.animate({
+			fit: {
+				eles: node,
+				padding: 230,
+			}
+		});
+		node.animate({
+			css: { 'width': nodeSize, 'height' : nodeSize}
+		});
 	}
-	
 })
 
 </script>
+<script type="text/javascript">
+        $(function() {
+            var offset = $("#graph").offset();
+            var topPadding = 15;
+            $(window).scroll(function() {
+                if ($(window).scrollTop() > offset.top) {
+                    $("#graph").stop().animate({
+                        marginTop: $(window).scrollTop() - offset.top + topPadding
+                    });
+                } else {
+                    $("#graph").stop().animate({
+                        marginTop: 0
+                    });
+                };
+            });
+        });
+        window.onresize = function(){ location.reload(); }
```

Comment:
does this reload the page as soon as the user resizes the window? the user would loose on whatever part he zoomed in on. that would be pretty annoying.